### ISSUE TITLE
Arrows are not positioning well with square waves

### ISSUE THREAD

**Issue by tehKaiN** [drom/wavedrom#136]
Consider code:

`{ signal: [
  { name: 'A', wave: '0h..l..',  node: '.a..b..' },
],
  edge: ['a<->b']
}`

Dimension doesn't align its left arrow on rising edge of signal

**Comment by drom** [comment]
Yes. Dependency arrows were designed for the normal signal behavior, which is delayed related to the clock.
`h`, `l`, `p`, `n`,... clock transitions, are immediate and misaligned with node anchors.

**Comment by jason-s** [comment]
+1 -- I would like to use square waves also with timing indications
